src/app/api/common/delegates/getDelegates.ts

```diff
@@ -304,6 +304,7 @@ async function getDelegates({
       },
       citizen: delegate.citizen,
       statement: delegate.statement,
+      numOfDelegators: BigInt(delegate.num_of_delegators),
```

Comment:
So, the data for `Delegated from` column exists here. We just need to propagate it.

But data for the `For/Against/Abstain` column does not exist. It only exists as part of the `getDelegate` request function in the same file. But for this function (`getDelegates`) it doesn't exist. Would someone be able to add `votedFor`, `votedAgainst`, etc as in the `getDelegate` function below? (And then I can sync to main and build my PR on top of that). Happy to do it too if I can get instructions. I tried to understand the code myself but the SQL here is way too complex and I'm afraid to break something